### PR TITLE
move away from deprecated consts

### PR DESCRIPTION
**Pull request by blizzz**
Sequel to https://github.com/owncloud/core/pull/15724

@MorrisJobke @Xenopathic @nickvergessen

### CODE REVIEW

File: apps/user_ldap/group_ldap.php [hunk 1]
```diff
@@ -746,6 +746,6 @@ public function groupExists($gid) {
 	* compared with OC_USER_BACKEND_CREATE_USER etc.
 	*/
 	public function implementsActions($actions) {
-		return (bool)(OC_GROUP_BACKEND_COUNT_USERS & $actions);
+		return (bool)(\OC_Group_Backend::COUNT_USERS & $actions);
```
Comment (nickvergessen) [inline apps/user_ldap/group_ldap.php]: Now when I see this, I think the constants should have been added to the public class instead: https://github.com/owncloud/core/blob/master/lib/public/groupinterface.php

**Comment by nickvergessen** [comment]
:+1:

File: apps/user_ldap/group_ldap.php (hunk 1 above)
Comment (DeepDiver1975) [inline apps/user_ldap/group_ldap.php]: Not really - this whole concept of implementsActions is no wrong. We need to cleanup this and use proper interfaces.

**Comment by DeepDiver1975** [comment]
:+1:

File: apps/user_ldap/group_ldap.php (hunk 1 above)
Comment (nickvergessen) [inline apps/user_ldap/group_ldap.php]: okay then its a "one day backlog" thing, I was mistaken anyway. I thought I moved them for 8.1, but they are moved in 8.0, so not worth the troubles anymore

**Pull request merged**